src/server/pps/pretty/pretty.go

```diff
@@ -186,6 +189,78 @@ Job Counts:
 	return nil
 }
 
+// PrintDatumInfoHeader prints a file info header.
+func PrintDatumInfoHeader(w io.Writer) {
+	fmt.Fprint(w, "ID\tSTATUS\tTIME\t\n")
+}
```

Comment:
Offline discussion -- the example output below though it only has 3 columns is quite wide. The hash in particular is long. It really should be shorter, but we've made that into a separate issue: https://github.com/pachyderm/pachyderm/issues/2113